docs/guide/Titanium_SDK/Titanium_SDK_Release_Notes/Titanium_SDK_Release_Notes_8.x/Titanium_SDK_8.0.2.GA_Release_Note.md

```diff
@@ -45,7 +45,7 @@ As of this GA release, the previous Titanium SDK patch release is no longer supp
 
 * [TIMOB-27101](https://jira.appcelerator.org/browse/TIMOB-27101) - Android: Window open() wrongly does a shared-element fade-in animation by default as of 8.0.1
 
-* [TIMOB-27102](https://jira.appcelerator.org/browse/TIMOB-27102) - iOS: "appc run" wrongly edits a Titanium SDK installed file causing studio to beachball and rebuild its workspace
+* [TIMOB-27102](https://jira.appcelerator.org/browse/TIMOB-27102) - iOS: "ti build" wrongly edits a Titanium SDK installed file causing studio to beachball and rebuild its workspace
```

Comment:
you'll get the same merge conflicts as in the other ticket with those lines :wink: It has a jira link that will be changed in the other PR

Reply:
but I already merged the master. Or should I've to merge the Jira PR and solve conflicts then?